tests/run-internalbench.py

```diff
@@ -8,6 +8,13 @@
 from glob import glob
 from collections import defaultdict
 
+import importlib
+
+run_tests_py = importlib.import_module("run-tests")
```

Comment:
Suggest using the pattern that other test runners use:
```py
run_tests_module = __import__("run-tests")
```
(No dependency on `importlib`.)